packages/react-app/.env.example

```diff
@@ -1,8 +1,14 @@
-REACT_APP_INFURA_ID=YOUR_INFURA_ID
-REACT_APP_API_KEY=
-REACT_APP_AUTH_DOMAIN=
-REACT_APP_PROJECT_ID=
-REACT_APP_STORAGE_BUCKET=
-REACT_APP_MESSAGING_SENDER_ID=
-REACT_APP_APP_ID=
-REACT_APP_MEASUREMENT_ID=
\ No newline at end of file
+REACT_APP_INFURA_ID=# your infura id
+REACT_APP_API_KEY=# your firebase api key
+REACT_APP_AUTH_DOMAIN=# your firebase auth domain
+REACT_APP_PROJECT_ID=# your firebase project id
+REACT_APP_STORAGE_BUCKET=# your firebase storage bucket
+REACT_APP_MESSAGING_SENDER_ID=# your firebase messaging sender id
+REACT_APP_APP_ID=# your firebase app id
+REACT_APP_MEASUREMENT_ID=# your firebase measurement id
+REACT_APP_CLIENT_ID=# your unstoppable domain client id
+REACT_APP_CLIENT_SECRET=# your unstoppable domain client secret
+REACT_APP_REDIRECT_URIS=# your unstoppable domain redirect uris
+REACT_APP_DOMAIN_NAME=# your domain name example: http://localhost:3000
```

Comment:
```
REACT_APP_CLIENT_ID=# your unstoppable domain client id
REACT_APP_CLIENT_SECRET=# your unstoppable domain client secret
REACT_APP_REDIRECT_URIS=# your unstoppable domain redirect uris
```

@chunza2542 if you need this for testing please DM me on discord spiritbro1#1101 or email me 
on 3aml5sclu@mozmail.com will give you that value for environment variable